frontend/src/config.ts

```diff
@@ -1,5 +1,6 @@
 export default {
     socure: {
-        api_key: 'adcd2fe2-b49c-4d20-ba72-4fa582c2c53d'
+        api_key: '',
```

Comment:
I am not sure if these keys are public? I guess they are since they are on the frontend.